code/packaging/app-config/src/main/resources/META-INF/cattle/iaas-api/defaults.properties

```diff
@@ -73,7 +73,7 @@ api.request.options=_plainId,_role,_actionLinks
 console.url.format.websocket-vnc=${static}/vnc.html?path=${path}&host=${host}&port=${port}&password=${password}&autoconnect=1
 
 api.proxy.allow=true
-api.proxy.whitelist=*.amazonaws.com,*.amazonaws.com.cn,forums.rancher.com,api.exoscale.ch,api.ubiquityhosting.com,api.digitalocean.com
+api.proxy.whitelist=*.amazonaws.com,*.amazonaws.com.cn,forums.rancher.com,api.exoscale.ch,api.ubiquityhosting.com,api.digitalocean.com,*.t-systems.com
```

Comment:
I think this can safely be `*.otc.t-systems.com`, (Open Telekom Cloud), their entire domain is a little broad.